### PR TITLE
Patch: limit sidebar space-between to Welcome items

### PR DESCRIPTION
**Pull request by godfreyyeung**

### CODE REVIEW

File: src/components/SidebarContent/SidebarContent.tsx [hunk 1]
```diff
@@ -77,13 +77,13 @@ export const SidebarContent = () => {
   }
 
   return (
-    <>
+    <Flex height="100%" direction="column" justify="space-between">
       <Box padding="2rem 1rem" paddingBottom="0rem">
```
[the remark this answers is not in the document]
Comment (godfreyyeung) [reply src/components/SidebarContent/SidebarContent.tsx:81]: This is what I'm seeing, if the screen height and width are a bit bigger. 
image

[the remark this answers is not in the document]
Comment (horatiorosa) [reply src/components/SidebarContent/SidebarContent.tsx:81]: No, underneath the welcome content. There is no space between `their homes or neighborhoods.` and `*Community Districts are approximated using data from Public Use Microdata Areas (PUMAs).` . Take a look and compare it against the DIsplacement Risk Map view:
Screen Shot 2022-04-13 at 1 10 58 PM

[the remark this answers is not in the document]
Comment (godfreyyeung) [reply src/components/SidebarContent/SidebarContent.tsx]: I see, good catch. I made an update, can you re-review?

**Review by horatiorosa — APPROVED** [review]
ship it

**Pull request merged**